packages/storybook-a11y-test/README.md

```diff
@@ -49,36 +49,39 @@ For optimal test performance, break the build and accessibility testing steps in
 Make sure you have built your Storybook instance and there is an `iframe.html` file that you can point the test towards.
 
 ```js
-const {testPages, getCurrentStoryIds} = require('@shopify/storybook-a11y-test');
+const {A11yTests} = require('@shopify/storybook-a11y-test');
 
 (async () => {
   // Custom URL or file path to Storybook’s iframe
   const iframePath = `file://${__dirname}/../build/storybook/static/iframe.html`;
 
-  // Grab all Story IDs
-  const storyIds = await getCurrentStoryIds({
-    iframePath,
-
-    // Optional, IDs of stories that shouldn’t be tested (for example: playgrounds)
-    skippedStoryIds: [],
-  });
-
-  // Run tests on all stories in `storyIds`
-  const results = await testPages({
-    iframePath,
-    storyIds,
-
-    // Optional: maximum time in milliseconds to wait for the browser instance to start.
-    // Defaults to 30000 (30 seconds). Pass 0 to disable timeout.
-    timeout: 30000,
-  });
-
-  if (results.length) {
-    console.error(`‼️  Accessibility violations found`);
-    console.log(results.join('\n'));
-    process.exit(1);
-  } else {
-    console.log('🧚  Accessibility tests passed');
+  const tests = new A11yTests(iframePath);
+
+  try {
+    // Grab all Story IDs
+    const storyIds = await tests.currentStoryIds({
```

Comment:
Story IDs will be able to be accessed via the json file exported at build time in newer versions of storybook. Eventually we won't need this. But that can be a separate PR. I just think it's worth mentioning here in case that changes your design.